model/src/main/kotlin/utils/OrtResultExtensions.kt

```diff
@@ -84,3 +91,38 @@ fun OrtResult.getDetectedLicensesWithCopyrights(
     omitExcluded: Boolean = true
 ): Map<String, Set<String>> =
     LicenseResolver(this, packageConfigurationProvider).getDetectedLicensesWithCopyrights(id, omitExcluded)
+
+/**
+ * Create [PackageConfiguration]s for the projects in this [OrtResult]. [PathExclude]s and [LicenseFindingCuration]s are
+ * taken from the [RepositoryConfiguration]. The [Provenance] is taken from [Project.vcsProcessed]. Projects with a
+ * blank VCS URL or revision are ignored, because no [VcsMatcher] can be created for them.
+ */
+fun OrtResult.getProjectConfigurations(): List<PackageConfiguration> {
+    fun VcsInfo.toMatcher() =
+        if (url.isBlank() || revision.isBlank()) {
+            null
+        } else {
+            VcsMatcher(
+                type = type,
+                url = url,
+                revision = revision,
```

Comment:
PackageConfigurations for Packages always have to use resolvedRevision, should we stick to it here as well for consistency? ..or is it maybe even required?

Reply:
Projects do not have a resolved revision, they just take the revision from the working tree. The resolved revision is only added by the downloader and scanner.